src/common.js.ts

```diff
@@ -252,89 +252,96 @@ function getErrorOrWarningFull(issue: OperationOutcomeIssue) {
-        if (issue.diagnostics.includes('LOINC is not indexed!')) return false;
         if (issue.diagnostics.includes('Error HTTP 403 Forbidden validating CodeableConcept')) return false;
-
-        if (issue.diagnostics.includes('Code system https://dmd.nhs.uk/ could not be resolved.')) return false
-/*
-        if (issue.diagnostics.includes('http://snomed.info/sct')) {
-            if (issue.diagnostics.includes('https://fhir.hl7.org.uk/ValueSet/UKCore-MedicationCode')) return false
-            if (issue.diagnostics.includes('https://fhir.hl7.org.uk/ValueSet/UKCore-VaccineCode')) return false
-        }
-*/
-        if (issue.diagnostics.includes('Unknown code')) {
-            if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/NHSDigital-SDS-JobRoleCode')) return false
-        }
+        
         if (issue.diagnostics.includes('None of the codings provided are in the value set')) {
             if (issue.diagnostics.includes('https://fhir.nhs.uk/CodeSystem/NHSDigital-SDS-JobRoleCode')) return false
             if (issue.diagnostics.includes('http://snomed.info/sct')) {
                 // Not defined in UKCore and valueset is extensible
                 return !issue.diagnostics.includes('http://hl7.org/fhir/ValueSet/observation-methods');
-
             }
         }
-        if (issue.diagnostics.includes('must be of the format')) {
-            return true;
-        }
-        
-        if (issue.diagnostics.includes('Inappropriate CodeSystem URL') && issue.diagnostics.includes('for ValueSet: http://hl7.org/fhir/ValueSet/all-languages')) {
-            return false
-        }
+        */
     }
 
-    // TODO this needs to be turned to true 1/8/2022 Warnings not acceptable on NHS Digital resources
-
+    // COMMENT WAS: TODO this needs to be turned to true 1/8/2022 Warnings not acceptable on NHS Digital resources
+    // Actual comment is: if error not handled above, return error if FailOnWarning is true 
     return failOnWarning;
 }
 function raiseError(issue: OperationOutcomeIssue) : boolean {
     if (issue != undefined) {
         if (issue.diagnostics != undefined) {
-            // List of errors to ignore
-            if (issue.diagnostics.includes('could not be resolved, so has not been checked')) return false;
-
+            // List of errors to ALWAYS ignore
+            
+            // languages, known issue!
+            if (issue.diagnostics.includes('Inappropriate CodeSystem URL') && issue.diagnostics.includes('for ValueSet: http://hl7.org/fhir/ValueSet/all-languages')) {
+                return false
+            }
+            
             // Ignore LOINC Errors for now
             if (issue.diagnostics.includes('http://loinc.org')) return false;
+            
+            // ignore dm+d / read errors
+            if (issue.diagnostics.includes('Code system https://dmd.nhs.uk/ could not be resolved.')) return false
```

Comment:
dm+d is in ontoserver sho should resolve. I commented this line and line 269 out on my fork and no additional errors showed in the log https://github.com/ryma2fhir/RYAN-FHIR-R4-UKCORE-STAGING-MAIN/actions/runs/7422530696